packages/affiliate/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "automattic/jetpack-affiliate",
+	"description": "Reads an affiliate code for Jetpack. Useful to add it to the connection URL so it's read by wpcom.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"automattic/jetpack-constants": "@dev",
+		"automattic/jetpack-status": "@dev"
```

Comment:
We don't seem to be using any of these 2 packages in this one. Remove and update composer.lock?